pkg/specgenutil/specgen.go

```diff
@@ -622,7 +622,12 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *entities.ContainerCreateOptions
 		if opt == "no-new-privileges" {
 			s.ContainerSecurityConfig.NoNewPrivileges = true
 		} else {
-			con := strings.SplitN(opt, "=", 2)
+			var con []string
+			if strings.Contains(opt, "=") {
+				con = strings.SplitN(opt, "=", 2)
+			} else if strings.Contains(opt, ":") {
```

Comment:
It may be worth leaving some breadcrumbs here in form of a code comment: Docker deprecated the ":" syntax but still supports it. So we need to as well.